src/CultureContext.js

```diff
@@ -5,12 +5,14 @@ class CultureContext extends Component {
   static childContextTypes = {
     culture: React.PropTypes.string,
     translations: React.PropTypes.object,
+    localeFallback: React.PropTypes.bool
```

Comment:
I don't think we should provide this option, I think 'fallback' should be always on by design, with no way to turn it off.